pkgs/by-name/ks/ksh93/package.nix

```diff
@@ -0,0 +1,59 @@
+{
+  fetchFromGitHub,
+  lib,
+  stdenv,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "ksh";
+  version = "1.0.10-unstable-2025-10-07";
+
+  src = fetchFromGitHub {
+    owner = "ksh93";
+    repo = "ksh";
+    rev = "ab6f3c1b0219e2e6afbd0cc4ef67e1a6293c471b";
+    hash = "sha256-9ZwkH7llfs67PVMbn/8bYsLSIV+1GzOYwXpKchGLdKk=";
+  };
+
+  buildPhase = ''
+    runHook preBuild
+    ./bin/package make
+    runHook postBuild
+  '';
+
+  doCheck = true;
+  checkPhase = ''
+    runHook preCheck
+    ./bin/package use
+    runHook postCheck
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    ./bin/package install $out
+    runHook postInstall
+  '';
+
+  postFixup = lib.optionalString stdenv.hostPlatform.isDarwin ''
```

Comment:
Can this use `fixDarwinDylibNames`?

Reply:
Yes, it'll fix the install name of the dylib, but not the binaries itself so this piece of postFixup code is still required I think.